client/GlobalStyles.js

```diff
@@ -4,7 +4,7 @@ const GlobalStyles = createGlobalStyle`
     body{
         margin:0;
         padding:0;
-        font-size: 62.5%;
+        font-size: 1rem;

```

Comment:
use fosnt size 1 rem on your local styles

Reply:
@melliottgithub corrected this and added local style